azure-pipelines.yml

```diff
@@ -89,6 +89,9 @@ jobs:
           #----------------------------------------------------------------------
           # Install all tools and check configuration
           #----------------------------------------------------------------------
+          - bash: exit 0
+            condition: or(contains(variables['Build.SourceVersionMessage'], '[skip ci]'), contains(variables['Build.SourceVersionMessage'], '[ci skip]'))
+            displayName: 'CHECK: check if commit contain CI skip'
```

Comment:
commit contains ? (with 's')

Reply:
Yes I believe it is; https://docs.microsoft.com/en-us/azure/devops/pipelines/process/expressions?view=azure-devops :smile:

Reply:
And here's an example; https://github.com/Microsoft/azure-pipelines-agent/issues/858#issuecomment-457027046

Reply:
I just talk about the message we display, not about the syntax lol 😂

Reply:
Ah, got it. Yes, contains is I think the correct form. :smile: